CHANGELOG.md

```diff
@@ -1,10 +1,42 @@
 # Azure Kubernetes Service Changelog
 
+## Release 2023-10-15
+
+Monitor the release status by regions at [AKS-Release-Tracker](https://releases.aks.azure.com/).
+
+### Announcements
+
+* No new clusters can be created with [Azure AD Integration (legacy)](https://learn.microsoft.com/azure/aks/azure-ad-integration-cli). Existing AKS clusters with Azure Active Directory integration will keep working. All Azure AD Integration (legacy) AKS clusters will be migrated to [AKS-managed Azure AD](https://learn.microsoft.com/azure/aks/managed-azure-ad) automatically starting from 1st Dec. 2023. We recommend updating your cluster with AKS-managed Azure AD before 1 Dec 2023. This way you can manage the API server downtime during non-business hours.
+* [CVE-2023-29332](https://github.com/Azure/AKS/issues/3904) Please update your AKS VHD to at least VHD version 230801 as mentioned in the issue
+
+### Release notes 
+* Feature 
```

Comment:
Missing above callout and the ones carried over from last week on probes, that now are out in docs.

@AllenWen-at-Azure @chasewilson

Reply:
Kevin added this item below, might to refine it from "Customize the load balancer health probe." to "AKS supports to use annotations to configure the load balancer health probe for different service ports." @chasewilson @palma21 thoughts?